CONTRIBUTING.md

```diff
@@ -29,6 +29,8 @@ If there is no pre-existing issue or it has been closed, please click on the "Ne
 
 If you do not receive a reply to your new issue or comment in a timely manner, please email support@dataverse.org with a link to the issue.
 
+We are aware of the new issue templates described at https://help.github.com/articles/about-issue-and-pull-request-templates but haven't converted over yet.
```

Comment:
My friend @ysuarez said recently that he'd be happy to make a pull request to switch us over to the new style. As part of this we'd delete this link and revisit the text below.

Reply:
@ysaurez, pull requests welcome! If you have something in mind feel free to create the issue and PR and we can work with you on it.

Reply:
Created https://github.com/IQSS/dataverse/issues/6227